src/integrations/Mixpanel/browser.js

```diff
@@ -23,6 +23,7 @@
 /* eslint-disable no-underscore-dangle */
 /* eslint-disable class-methods-use-this */
 import get from 'get-value';
+import isEmpty from 'lodash.isempty';
```

Comment:
why add a new lodash method? cant we just use !isNotEmpty instead?

Reply:
Removed the lodash import